docs/config.md

```diff
@@ -100,7 +100,7 @@ The new group labels do not need to be a part of your source `labels` list.
 #### Example
 ```yaml
 grouped-labels:
-  ill: [insomnia, chickenpox, ebola]
+  animal: [dog, cat, fox]
```

Comment:
Just changing to a less charged example.

Reply:
just saying: https://en.wikipedia.org/wiki/List_of_fictional_diseases

Reply:
OK but cooties is on that list and that's definitely real, so...